samples/subsys/lorawan/class_a/src/main.c

```diff
@@ -31,8 +31,9 @@ static void dl_callback(uint8_t port, uint8_t flags,
 			int16_t rssi, int8_t snr,
 			uint8_t len, const uint8_t *data)
 {
-	LOG_INF("Port %d, Pending %d, RSSI %ddB, SNR %ddBm",
-			port, flags & LORAWAN_DATA_PENDING, rssi, snr);
+	LOG_INF("Port %d, Pending %d, RSSI %ddB, SNR %ddBm Time %d",
+			port, flags & LORAWAN_DATA_PENDING, rssi, snr,
+			flags & LORAWAN_TIME_UPDATED);
```

Comment:
Flags prints should probably be done with `!!(flags & LORAWAN_TIME_UPDATED)`, otherwise the text will be `0` or `2`